src/operator/controllers/kafkaserverconfig_controller.go

```diff
@@ -332,7 +335,7 @@ func (r *KafkaServerConfigReconciler) uploadKafkaServerConfigs(ctx context.Conte
 }
 
 func kafkaServerConfigCRDToCloudModel(kafkaServerConfig otterizev1alpha2.KafkaServerConfig) (graphqlclient.KafkaServerConfigInput, error) {
-	var topics []graphqlclient.KafkaTopicInput
+	topics := make([]graphqlclient.KafkaTopicInput, 0)
```

Comment:
Fixes bug where if no `topics` were specified in the spec, the cloud reconciler would fail to report this because it was `nil` (empty list is ok, nil is not)